### PR TITLE
1.26 is out of support

### PR DESCRIPTION
**Pull request by kannon92**
Also dropped a 1.25 that was hanging around..

cc @alculquicondor @mimowo @tenzen-y 

I was doing this for jobset and wanted to open it up for Kueue.

### CODE REVIEW

File: config/jobs/kubernetes-sigs/kueue/kueue-presubmits-release-0-4.yaml
Comment (alculquicondor) [inline config/jobs/kubernetes-sigs/kueue/kueue-presubmits-release-0-4.yaml:1]: Please remove this file instead.

Comment (kannon92) [reply config/jobs/kubernetes-sigs/kueue/kueue-presubmits-release-0-4.yaml:1]: done.

**Review by alculquicondor — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**